### PR TITLE
Add task doc for chef

### PR DESCRIPTION
**Pull request by jcreixell**
- Adds basic instructions on how to install Grafana Agent using Chef.
  - Fixes https://github.com/grafana/agent/issues/5961

### CODE REVIEW

**Comment by jcreixell** [comment]
follows the pattern from #6129, already incorporated some of the feedback from the PR.

I have tested this with Fedora and Ubuntu, the full cookbook used for testing can be found [here](https://github.com/jcreixell/grafana-agent-cookbook/blob/main/cookbooks/grafana-agent/recipes/default.rb).

File: docs/sources/flow/get-started/install/chef.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+---
+
+# Install or uninstall {{% param "PRODUCT_NAME" %}} using Chef
+
+You can use Chef to install and manage {{< param "PRODUCT_NAME" >}}.
+
+## Before you begin
+
+- These steps assume you already have a working [Chef](https://www.chef.io/products/chef-infrastructure-management/) setup.
+- You can add the resources below to any new or existing recipy you choose.
```
Comment (clayton-cornell) [inline docs/sources/flow/get-started/install/chef.md:18]:
```suggestion
- You can add the following resources to any new or existing recipe you choose.
```

File: docs/sources/flow/get-started/install/chef.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+- These tasks install {{< param "PRODUCT_NAME" >}} from the package repositories. They expect to target Linux systems from the following families:
```
Comment (clayton-cornell) [inline docs/sources/flow/get-started/install/chef.md:19]:
```suggestion
- These tasks install {{< param "PRODUCT_NAME" >}} from the package repositories. The tasks target Linux systems from the following families:
```

**Review by clayton-cornell — COMMENTED** [review]
Some doc review input

File: docs/sources/flow/get-started/install/chef.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  - Debian (including Ubuntu)
+  - RedHat Enterprise Linux
+  - Amazon Linux
+  - Fedora
+
+## Steps
+
+To add {{% param "PRODUCT_NAME" %}} to a host:
+
+1. Add the following resources to a recipe to add the Grafana package repositories to your system:
```
Comment (thampiotr) [inline docs/sources/flow/get-started/install/chef.md:29]:
```suggestion
1. Add the following resources to your [Chef][] recipe to add the Grafana package repositories to your system:
```

File: docs/sources/flow/get-started/install/chef.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+aliases:
+- /docs/grafana-cloud/agent/flow/get-started/install/chef/
+canonical: https://grafana.com/docs/agent/latest/flow/get-started/install/chef/
+description: Learn how to install Grafana Agent Flow with Chef
+menuTitle: Chef
+title: Install Grafana Agent Flow with Chef
+weight: 550
+---
+
+# Install or uninstall {{% param "PRODUCT_NAME" %}} using Chef
```
Comment (thampiotr) [inline docs/sources/flow/get-started/install/chef.md:11]:
```suggestion
# Install {{% param "PRODUCT_NAME" %}} with Chef
```

**Review by thampiotr — APPROVED** [review]
Looks good to me! Couple of suggestions only.

**Pull request merged**